opmd_viewer/openpmd_timeseries/data_reader/utilities.py

```diff
@@ -124,3 +124,34 @@ def get_shape(dset):
         shape = dset.shape
 
     return(shape)
+
+
+def join(base_path, *paths):
+    """
+    Join path components using '/' as separator.
+    This method is defined as an alternative to os.path.join, which uses '\\'
+    as separator in Windows environments and is therefore not valid to navigate
+    within data files.
+
+    Parameters:
+    -----------
+    base_path: string cointaining the base path
+
+    *paths: all other strings with path components to join
+
+    Returns:
+    --------
+    A string with the complete path using '/' as separator.
+    """
+    # Initialize path
+    path = base_path
+    # Join all path components adding a '/'
+    try:
+        for p in paths:
+            path += '/' + p
+    except:
+        print("Error joining strings.")
```

Comment:
I think that the above code could be made a bit more concise:
```python
def filepath_join( *paths ):
     """
     Copy previous docstring... but without base_path
     """
     path = '/'.join(paths)
     path = path.replace('//', '/')
     return path
```
Would you be okay with using the above version in your code?

Reply:
Sounds good. Or maybe even `get_infile_path`.

Reply:
`join_infile_path`?

Reply:
I like this last name. I just submitted a new commit including both changes.